doc/unicode.md

```diff
@@ -11,12 +11,19 @@ Notes on Unicode in Shell
 
 ## Philosophy
 
-Oils is UTF-8 centric, unlike `bash` and other shells.
+Oils is UTF-8 and JSON centric, unlike `bash` or other C-string and UNIX centric shells.
```

Comment:
the JSON part doesn't really make sense; it is potentially confusing because JSON has elements of both UTF-8 and UTF-16